setup.py

```diff
@@ -20,7 +20,7 @@
 long_description_content_type = 'text/markdown'
 
 # this grabs the requirements from requirements.txt
-REQUIREMENTS = [i.strip().split('==')[0] for i in open(os.path.join(here, "requirements.txt")).readlines()]
+REQUIREMENTS = [i.strip() for i in open(os.path.join(here, "requirements.txt")).readlines()]
```

Comment:
Pinning deps is ideal, but this results in incompatibilities with different Python versions. I will test this.